Update howto-create-a-storageclass.md (#87923)
Azure/azure-content: articles/openshift/howto-create-a-storageclass.md

oc create -f azure-storageclass-azure-file.yaml
```

> Mount options for Azure Files will generally be dependent on the workload that you are deploying and what the application depends on. You can get a lot of details regarding the various mount options available here: https://linux.die.net/man/8/mount.cifs Specifically for Azure files, we do have some additional mandatory and recommended parameters that you should consider using.

Reviewer: ```suggestion
Mount options for Azure Files will generally be dependent on the workload that you are deploying and the requirements of the application. Specifically for Azure files, there are additional parameters that you should consider using.
```

Author: In general ok with your suggestion, but there seems to be an "are" missing after "there".

Reviewer: Thank you for spotting that!

> Mandatory parameters: 
> - "mfsymlinks" to map symlinks to a form the client can use
> - "noperm" to disable permission checks on the client side 
> 
> Recommended parameters: 
> - "nossharesock" to disable reusing sockets if the client is already connected via an existing mount point.
> - "actimeo=30" (or higher) to increase the time the CIFS client caches file and directory attributes
> - "nobrl" to disable sending byte range lock requests to the server. For application which have challenges with posix locks.

Reviewer 2: ```suggestion
Mandatory parameters: 
- "mfsymlinks" to map symlinks to a form the client can use
- "noperm" to disable permission checks on the client side

Recommended parameters: 
- "nossharesock" to disable reusing sockets if the client is already connected via an existing mount point
- "actimeo=30" (or higher) to increase the time the CIFS client caches file and directory attributes
- "nobrl" to disable sending byte range lock requests to the server and for applications which have challenges with posix locks
```
consistent punctuation and grammar tweak
